src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -178,27 +169,21 @@ public static void setDegreesFromGearRatioRelativeEncoder(
 
   /** True if the arm is at the queried position. */
   public boolean atDesiredArmPosition() {
-    double armMarginDegrees =
-        desiredPosition == ArmPosition.STARTING
-            ? ArmCal.ARM_START_MARGIN_DEGREES
-            : ArmCal.ARM_MARGIN_DEGREES;
+    double armMarginDegrees = ArmCal.ARM_MARGIN_DEGREES;
     double armPositionToCheckDegrees = armPositionMap.get(desiredPosition);
     double armPositionDegrees = armEncoder.getPosition();
     return Math.abs(armPositionDegrees - armPositionToCheckDegrees) <= armMarginDegrees;
   }
 
   @Override
-  public void periodic() {
-    if (cancelScore && isScoring) {
-      goToPosition(ArmPosition.STARTING);
-      cancelScore = false;
-      isScoring = false;
-    }
+  public void periodic(){
+    
   }
 
-  /** Cancellation function */
-  public void cancelScore() {
-    cancelScore = true;
+  /**Cancellation function */
+  public void cancelScore(){
+      goToPosition(ArmPosition.STARTING);
+      isScoring = false;
```

Comment:
Is the plan for the FinishScore to check isScoring or is the plan to check isScoring then not run FinishScore at all if it's false?